src/System.IO.FileSystem/src/System/IO/Win32FileSystem.cs

```diff
@@ -368,7 +368,7 @@ public override DateTimeOffset GetCreationTime(string fullPath)
                 throw Win32Marshal.GetExceptionForWin32Error(errorCode, fullPath);
 
             long dt = ((long)(data.ftCreationTimeHigh) << 32) | ((long)data.ftCreationTimeLow);
-            return DateTimeOffset.FromFileTime(dt);
+            return DateTimeOffset.FromFileTime(dt).ToLocalTime();
```

Comment:
ToLocalTime() is unnecessary for FromFileTime, you are correct.

I missed CreationTime in my undoing of the ToLocalTime additions; I'll rebase again tomorrow morning and fix this.